aten/src/ATen/Declarations.cwrap

```diff
@@ -2083,55 +2083,6 @@
     - arg: THTensor* self
     - arg: THTensor* tensor
 ]]
-[[
-  name: _th_tril
-  cname: tril
```

Comment:
Does this mean we can delete the old TH code now? i.e. `THTensor_(tril)`

Reply:
I have removed tril. triu is required for a Lapack function, and both tril and triu for THC are also required for certain Magma ops.